README.md

```diff
@@ -41,6 +41,16 @@ must be applied via [patches](patches). See [below](#patches) for more info on p
 
 ## Updating an External Component to a Newer Version
 
+When updating a component that is used in both Arcade and another target repository, follow these steps:
+
+1. **Add the updated component**: Include the component with the updated version as a new submodule in source-build-externals.
+2. **Update the target repository**: Allow the updated version to flow to the target repository and update the target repository to use this new version.
+3. **Build and release**: Perform a build and release the artifacts to update the n-1 (previous) artifacts for Arcade.
+4. **Update Arcade**: Update Arcade to use the new version of the component.
+5. **Clean up**: Remove the outdated component from source-build-externals.
```

Comment:
I would put this entire section at the end of the "Updating an External Component to a Newer Version" heading and place it under a sub-heading named something like "Workflow for Repo Dependencies from Arcade".

Reply:
Moved to new subsection in https://github.com/dotnet/source-build-externals/pull/280/commits/3c856a4815ff8133ec423b50ef6975d97aee6f2c. I went with a title of "Updating an External Component Used in a Toolset Repo and in a Target Repo". It's a bit long and convoluted IMO, but I was having trouble thinking of a better title. Let me know if you have any suggestions.